script/ghIssues.pl

```diff
@@ -437,6 +435,100 @@ sub get_mentions {
     }
 }
 
+sub update_pr_template {
+    my ($data, $pr_number, $ia) = @_;
+
+    # find dax comment at spot #1 or bail
+    my @comments = $gh->issue->comments($pr_number);
+
+    my $comment_number;
+    if(scalar @comments){
+        my $comment = $comments[0];
+        return unless $comment->{user}->{login} eq 'daxtheduck';
+
+        # skip dax welcome message if it exists
+        if($comment->{body} =~ /Thanks for taking the time to contribute!/){
+            if(scalar @comments > 1){
+                $comment = $comments[1];
+                return unless $comment->{user}->{login} eq 'daxtheduck';
+                $comment_number = $comment->{id};
+            }
+        }else{
+            $comment_number = $comment->{id};
+        }
+    }
+
+    my $examples = $data->{example_query};
+
+    if(exists $ia->{other_queries}){
+        $ia->{other_queries} =~ s/"|\[|\]//g;
+        $examples .=", ". $ia->{other_queries};
+    }
+
+    my $browsers;
+    foreach my $browser (qw(safari firefox ie opera chrome)){
+        my $val = $ia->{"browsers_$browser"};
+        if($val){
+            $val = '&#10003;';
+        }else{
+            $val = ' ';
+        }
+        $browsers .= '['.$val.'] ' . $browser. "\n";
+    }
+
+    my $other_tests;
+    foreach my $test (qw(code_review design_review)){
```

Comment:
@jdorweiler Not sure this is actually helpful. Code review and design review don't necessarily exclude each other, and when a user submits a PR he won't have had any design or code review yet anyways.
Also we aren't using this info on IA Pages, nor we have specific plans about it for the near future.